libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1508,6 +1508,39 @@ bool ExpressionCompiler::visit(MemberAccess const& _memberAccess)
 			);
 			m_context << Instruction::BALANCE;
 		}
+		else if (member == "code")
+		{
+			// Stack: <address>
+			utils().convertType(
+				*_memberAccess.expression().annotation().type,
+				*TypeProvider::address(),
+				true
+			);
+
+			m_context << Instruction::DUP1 << Instruction::EXTCODESIZE;
```

Comment:
I'm always wondering, wo actually need to perform cleanup before such operations? Doesn't extcodesize just ignore the upper bytes?

Reply:
All opcodes using address do that, but shouldn't we leave the knowledge to the optimiser?

Reply:
Yep, just a thought.